Data.py

```diff
@@ -578,7 +578,10 @@ def create_netcdf_file(nc_file, longitude, latitude, **data_vars):
     # Define the dimensions
     rows, cols = latitude.shape
     # Assuming all data variables have the same 'time' dimension size
-    doy = next(iter(data_vars.values())).shape[2]
+    if next(iter(data_vars.values())).ndim == 1:
```

Comment:
@SubinKim-phy We actually do not need "doy" variable for data if it is not a 3d array data. Please be more creative to modify this code.